src/scripts.js

```diff
@@ -5,14 +5,18 @@ import './images/turing-logo.png'
 import Recipe from './classes/Recipe'
 import RecipeRepository from './classes/RecipeRepository'
 import recipeData from './data/recipes'
+// ------------------- GLOBAL VARIABLES
+const recipeRepo = new RecipeRepository(recipeData)
+let tag;
 
-// --------------------------------------------- QUERY SELECTORS
+// --------------------QUERY SELECTORS
 // const modalBtn = document.querySelector('#modalBtn');
 const modal = document.querySelector('#modal')
 const close = document.querySelector('#close')
 const recipeSection = document.querySelector('#recipeSection')
+const tags = document.querySelector('#tags')
```

Comment:
Are we using this query selector? I think I might be overlooking where it is used!

Reply:
@Adelle-Pitsas showed me where this is being used in the event listeners section!